ios/titanium.xcconfig

```diff
@@ -6,14 +6,9 @@
 //
 TITANIUM_SDK_VERSION = 9.0.0.v20200205142057
 
-
-// 
+//
 // THESE SHOULD BE OK GENERALLY AS-IS
 //
-TITANIUM_SDK = ~/Library/Application Support/Titanium/mobilesdk/osx/$(TITANIUM_SDK_VERSION)
-TITANIUM_BASE_SDK = "$(TITANIUM_SDK)/iphone/include"
-TITANIUM_BASE_SDK2 = "$(TITANIUM_SDK)/iphone/include/TiCore"
-TITANIUM_BASE_SDK3 = "$(TITANIUM_SDK)/iphone/include/ASI"
-TITANIUM_BASE_SDK4 = "$(TITANIUM_SDK)/iphone/include/APSHTTPClient"
-HEADER_SEARCH_PATHS = $(TITANIUM_BASE_SDK) $(TITANIUM_BASE_SDK2) $(TITANIUM_BASE_SDK3) $(TITANIUM_BASE_SDK4)
+TITANIUM_SDK = /Users/$(USER)/Library/Application Support/Titanium/mobilesdk/osx/$(TITANIUM_SDK_VERSION)
```

Comment:
@anil-shukla-axway, its better to separate config file changes into separate PR.

Reply:
@anil-shukla-axway , I agree with @fasihul-axway , Please address these changes in different PR

Reply:
Done, Please check PR - https://github.com/appcelerator-modules/ti.map/pull/296